Public/Src/Engine/Processes/Tracing/LogEventId.cs

```diff
@@ -59,7 +59,8 @@ public enum LogEventId
         PipProcessStandardInputException = 86,
 
         PipProcessInvalidErrorRegex = 89,
-        //// Reserved  = 90,

```

Comment:
>//// Reserved  = 90, [](start = 8, length = 20)

I suspect this is marked "reserved" because it is used in some other eventid file. So it probably isn't ok to reuse. If my suspicion is correct there should also be a unit test that fails due to the conflict.

Reply:
It seems it is the opposite. The one marked as reserved is available and those ones not marked may have been used. I did a full text search for some instance and always find the reserved ones only appear as reserved.